crates/sovereign-sdk/full-node/sov-stf-runner/src/config.rs

```diff
@@ -66,6 +66,8 @@ const fn default_batch_requests_limit() -> u32 {
 pub struct StorageConfig {
     /// Path that can be utilized by concrete rollup implementation
     pub path: PathBuf,
+    /// File descriptor limit for RocksDB
+    pub db_max_open_files: Option<i32>,
```

Comment:
I believe unix uses u64, but rocksdb accepts i32 since -1 has a special meaning: `Value -1 means files opened are always kept open`. Should we allow users to define such behavior through config?

Reply:
lets keep it this way